lib-multisrc/heancms/src/eu/kanade/tachiyomi/multisrc/heancms/HeanCms.kt

```diff
@@ -70,6 +76,37 @@ abstract class HeanCms(
         .add("Origin", baseUrl)
         .add("Referer", "$baseUrl/")
 
+    private val authHeaders: Headers by lazy {
+        val builder = headersBuilder()
+        if (preferences.loginPref && enableLogin) {
+            val token = getToken()
+            if (token.isNotBlank()) {
+                builder.add("Authorization", "Bearer $token")
+            }
+        }
+        builder.build()
+    }
+
+    private fun getToken(): String {
```

Comment:
Might be better to make this nullable so you don't rely on the string being blank elsewhere. Unless you have observed the response containing an empty string and want to handle that as well.
```suggestion
    private fun getToken(): String? {
```